### PR TITLE
uhhhhh words

### PR DESCRIPTION
**Pull request by 0xEmma**

### CODE REVIEW

File: lib/plugins/eval.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@ module Eval
   extend Discordrb::Commands::CommandContainer
 
   command(:eval) do |event, *code|
-    break unless event.user.id == CONFIG['owner_id']
+    break unless event.user.id == CONFIG['owner_id' || '484040243818004491']
```
Comment (Chew) [inline lib/plugins/eval.rb]: this isn't how ruby works

**Review by Chew — CHANGES_REQUESTED** [review]

**Comment by Chew** [comment]
This is epic merging now

**Comment by Chew** [comment]
Wait that's the wrong button

**Comment by Chew** [comment]
How do i merge a pr

**Review by Chew — APPROVED** [review]

**Comment by 0xEmma** [comment]
https://help.github.com/articles/merging-a-pull-request/

**Comment by Chew** [comment]
nope those arent't i

**Comment by Chew** [comment]
perfect

**Comment by 0xEmma** [comment]
wwe must murge

**Comment by Chew** [comment]
This has been fixed and a push will be arriving shortly. Enjoy!

**Comment by Chew** [comment]
This appears to be fixed, if it hasn't, please re-open the issue and explain why it is still broken.
Thanks for the report!

**Pull request merged**